src/styles.scss

```diff
@@ -1,4 +1,4 @@
-@import "~@angular/material/prebuilt-themes/pink-bluegrey.css";
+@import "theme.scss";
 
```

Comment:
(opt) I think that would be good idea to put here a common font-family like mat-table (Roboto, "Helvetica Neue", sans-serif)

Reply:
Done